charts/ccsm-helm/test/zeebe/service_test.go

```diff
@@ -66,3 +66,32 @@ func (s *serviceTest) TestContainerSetGlobalAnnotations() {
 	// then
 	s.Require().Equal("bar", service.ObjectMeta.Annotations["foo"])
 }
+
+func (s *serviceTest) TestExtraServices() {
+	// given
+	options := &helm.Options{
+		SetValues: map[string]string{
+			"zeebe.service.extraPorts[0].name":       "hazelcast",
+			"zeebe.service.extraPorts[0].protocol":   "TCP",
+			"zeebe.service.extraPorts[0].port":       "5701",
+			"zeebe.service.extraPorts[0].targetPort": "5701",
+		},
+		KubectlOptions: k8s.NewKubectlOptions("", "", s.namespace),
+	}
+
+	// when
+	output := helm.RenderTemplate(s.T(), options, s.chartPath, s.release, s.templates)
+	var service coreV1.Service
+	helm.UnmarshalK8SYaml(s.T(), output, &service)
+
+	// then
+	expectedPort := int32(5701)
+	expectedName := "hazelcast"
+	expectedTargetPort := int32(5701)
+	containers := service.Spec.Ports
```

Comment:
:x: I assume you mean ports here, the naming is a bit confusing

Reply:
@Zelldon  I assume you meant the name of the test? I fixed that just now :)

Reply:
I meant `containers := service.Spec.Ports` I had expected `ports := service.Spec.Ports` :shrug:

Reply:
ahh fixed. Still a little new to tests :)